mono/metadata/assembly-internals.h

```diff
@@ -34,7 +34,14 @@ typedef enum {
 G_ENUM_FUNCTIONS (MonoAssemblyNameEqFlags);
 
 void
-mono_assembly_name_free_internal (MonoAssemblyName *aname);
+mono_assembly_name_free_internal_slow (MonoAssemblyName *aname);
+
+static inline void
+mono_assembly_name_free_internal (MonoAssemblyName *aname)
+{
+	if (aname)
```

Comment:
Why does this has to be inline ?

Reply:
It does not have to be. I can undo that.